### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by esthercheran**

### CODE REVIEW

File: pages/handoff.en.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 As Figma currently provides no way to inspect plugin data as a viewer (we hope they do allow that at some point), you'd have to fall back to either letting your engineers use the plugin themselves and inspect layers, or use more traditional methods. One of those methods would be Redlining, where you'd create layers showing which tokens were applied to what layers, still containing token names so it would be clear what to apply when.
 
+<img src="/annotate.gif" alt="Image alt" />
```
Comment (six7) [inline pages/handoff.en.md:5]: This graphic isn't loading for me, also Alt text should be explicit

**Review by six7 — CHANGES_REQUESTED** [review]

File: pages/handoff.en.md (hunk 1 above)
Comment (esthercheran) [reply pages/handoff.en.md]: Done

**Comment by esthercheran** [comment]
@six7 have a look please!

File: pages/multi-inspect.en.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  alt="multi-inspect in action"
+  url="/multi-inspect.mov"
+/>
+
+## Deep inspect
+
+By selecting `Deep Inspect` you can see the tokens applied to all the children of the selected layer.
+<br/>
+<br/>
+<img src='./multi-inspect-deep-inspect.png' alt="Deep inspecting a layer in Figma" width="500" />
```
Comment (six7) [inline pages/multi-inspect.en.mdx:25]:
```suggestion


<img src='./multi-inspect-deep-inspect.png' alt="Deep inspecting a layer in Figma" width="500" />
```

File: pages/multi-inspect.en.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<br/>
+<br/>
+<img src='./multi-inspect-select-all.png' alt="Selecting all layers with a particular token" width="500" />
+
+## Selecting a single layer with a particular token
+
+On the side of the token you can see how many layers a particular token is applied to. By clicking on the layer icon, you get a list of the different layers and you can select any particular layer.
+
+<br/>
+<br/>
```
Comment (six7) [inline pages/multi-inspect.en.mdx:39]:
```suggestion


```

File: pages/multi-inspect.en.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+<img src='./multi-inspect-select-one.png' alt="Selecting a single layer with a token" width="500" />
+
+## Removing tokens
+
+You can select a single token or `Select all` tokens and click on `Remove selected` to remove the token/s from the layer/s.
+
+<br/>
+<br/>
+<img src='./multi-inspect-remove.png' alt="removing a token" width="500" />
```
Comment (six7) [inline pages/multi-inspect.en.mdx:48]:
```suggestion


<img src='./multi-inspect-remove.png' alt="removing a token" width="500" />
```

File: pages/multi-inspect.en.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Remapping tokens
+
+Click on the `v` icon next to the token to remap it to another token on all the layers that it is applied.
+
+<br/>
+<br/>
```
Comment (six7) [inline pages/multi-inspect.en.mdx:55]:
```suggestion


```

File: pages/sync/multi-file.en.mdx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  width="100%"
+  height="auto"
+  loop
+  playing
+  controls
+  alt="multi-file sync"
+  url="/multi-file.mov"
+/>
+
+In the `settings` tab select your remote sync in the `Token Storage` section.
```
Comment (six7) [inline pages/sync/multi-file.en.mdx:16]:
```suggestion
In the `Settings` tab select your remote sync in the `Token Storage` section.
```

File: pages/themes/themes-pro.en.mdx [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# Themes-Pro 
+
+If you are a pro user, you have access to the pro `Theme` feature.
```
Comment (six7) [inline pages/themes/themes-pro.en.mdx:3]:
```suggestion
If you are on a Pro plan, you have access to the `Theme` feature.
```

**Pull request merged**